packages/block-library/src/list-item/block.json

```diff
@@ -21,8 +21,8 @@
 	},
 	"supports": {
 		"className": false,
+		"__experimentalSelector": ".wp-block-list > li",
```

Comment:
Just noting this should have used the [selectors API](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-selectors/) which landed at the end of 2022, rather than the deprecated `__experimentalSelector`. 

For example: 

```json
{
    ...
    "selectors": {
        "root": ".wp-block-list > li"
    }
}
```

Reply:
I'm in the process of dusting off an old PR to add color support to the List Item block. I can switch this over to use `selectors` as part of that.

Reply:
thank you!

Reply:
Thanks for following up on this!